lib/tests/test_gpg.rs

```diff
@@ -33,6 +34,90 @@ lBqJFFP2PqzT0CRfh94HSIMM
 -----END PGP PRIVATE KEY BLOCK-----
 "#;
 
+static GPGSM_FINGERPRINT: &str = "8F08B1D7DDEEFA91C25F8681F6494FF2E03DEADE";
+
+static GPGSM_PRIVATE_KEY: &str = r#"Issuer ...: /CN=JJ/O=X509 Signing Test
+Serial ...: 4C2B796B46814D39
+Subject ..: /CN=JJ/O=X509 Signing Test
+    aka ..: someone@example.com
+Keygrip ..: E595CEA8D5E74F60DEDF96753E041F64157DFBF6
+
+-----BEGIN PKCS12-----
```

Comment:
I guess it's fine for a test object, but, really, no one should be using `pbewithSHAAnd40BitRC2-CBC` and `pbeWithSHAAnd3-KeyTripleDES-CBC` these days. This suggests tooling with poor defaults was used (such as old openssl). Perhaps this means that someone with the relevant expertise with the tools should provide some guidance how to avoid this.

Reply:
My experience as a distro packager says: tests with bad crypto inevitably break as upstream defaults get tightened. Please do fix this. Consider RFC 9500.